### DISCUSSION TITLE
Checks shown as failed due to prior failure but have been fixed

### DISCUSSION THREAD

**Discussion by jmeridth** [community/community#61584] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.123.0 (105244228)
Device: iPhone15,2 (iOS 16.5.1)
</details>

https://github.com/argoproj/argo-helm/pull/2182 shows as failed checks on mobile due to previous failure. 

![image](https://github.com/community/community/assets/35014/aa88444a-cfbf-492d-83b4-d59eb42dc962)

But green/success on web, as expected. 

![image](https://github.com/community/community/assets/35014/ca570c68-1fca-4a8b-93bc-00cc86f551f1)

**Comment by aconsuegra** [comment, marked as answer]
Hi @jmeridth, thanks for reaching out with your feedback. This is known issue already in our radar but we don't have an estimation for a fix, I'll bump our internal issue with your feedback, so hopefully we can work on a fix soon.
